include/seeds.history.hpp

```diff
@@ -239,6 +241,13 @@ CONTRACT history : public contract {
         uint128_t by_account_key() const { return (uint128_t(account.value) << 64) + key.value; }
       };
 
+      TABLE deferred_id_table {
+        uint64_t id;
+      };
```

Comment:
Can you have a table without primary key?

Reply:
In this case yes, because it is a singleton. As the table will only contain one value I thought it was a good idea to make it one.